README.md

```diff
@@ -1,2 +1,303 @@
+
+   ```bash
+   az group create ${RESOURCE_GROUP_AIB} --location centralus
+   ```
+
+1. **Clone this repo locally.** _Optional._
+
+   This will allow you to edit the `azuredeploy.parameters.json` file to include your specific values.
+
+   ```bash
+   git clone https://github.com/mspnp/aks-jumpbox-imagebuilder
+
+   cd aks-jumpbox-imagebuilder
+   ```
+
+1. **Deploy AIB service's Managed Identity and assign Azure RBAC roles.**
+
+   Now you'll deploy the AIB service's Managed Identity, assign it Azure RBAC permissions, and deploy your AKS jumpbox image template. This does not perform the image build, but simply gets the definition of the image deployed to azure as a managed resource, and sets up the AIB service's identity such that it can build the image in its transient compute resources.
+
+   Option 1: _Requires cloned repo._
+
+   Update the values in `azuredeploy.parameters.json` to align with your environment. Specifically you'll be setting parameter values with the target subnet (from Step 2 above) in which the image will be built from within, what RBAC roles the service's Managed Identity will receive, and where the built image resource will be distributed to.
+
+   ```bash
+   az deployment group create -g ${RESOURCE_GROUP_AIB} -f azuredeploy.json -p "@azuredeploy.parameters.json" -n aibaksjumpboximgtemplate
+
+   export RESOURCE_GROUP_IMAGE=$(az deployment group show -g ${RESOURCE_GROUP_AIB} -n aibaksjumpboximgtemplate --query 'properties.parameters.imageDestinationResourceGroupName.value' -o tsv)
+   ```
+
+   Option 2:
+
+   Deploy without parameters file, setting the values below as appropriate. The first our are target virtual network values, the next two are the custom role ids (or fallbacks if custom roles are not able to be used), and the final is the resource group you wish the built managed image resource to be deployed to.
+
+   ```bash
+   export RESOURCE_GROUP_VNET="rg-enterprise-networking-spokes"
+   export VNET_NAME="vnet-imagebuilder"
+   export SNET_NAME="snet-imagebuilder"
+   export VNET_LOCATION="eastus2"
+   export NETWORK_CONTRIBUTOR_ROLE=$(NETWORK_CONTRIBUTOR_ROLE:-4d97b98b-1d4f-4787-a291-c67834d212e7) # Use custom role, or default to extra permissive Network Contributor role
+   export IMAGE_CONTRIBUTOR_ROLE=$(IMAGE_CONTRIBUTOR_ROLE:-b24988ac-6180-42a0-ab88-20f7382dd24c)     # Use custom role, or default to extra permissive Contributor role
+   export RESOURCE_GROUP_IMAGE="rg-mycluster"
+
+   az deployment group create -g ${RESOURCE_GROUP_AIB} -f https://raw.githubusercontent.com/mspnp/aks-jumpbox-imagebuilder/main/azuredeploy.json -p buildInVnetResourceGroupName=${RESOURCE_GROUP_VNET} buildInVnetName=${VNET_NAME} buildInVnetSubnetName=${SNET_NAME} location=${VNET_LOCATION} imageBuilderNetworkingRoleGuid=${NETWORK_CONTRIBUTOR_ROLE} imageBuilderImageCreationRoleGuid=${IMAGE_CONTRIBUTOR_ROLE} imageDestinationResourceGroupName=${RESOURCE_GROUP_IMAGE} -n aibaksjumpboximgtemplate
```

Comment:
It was about the Azure Button, letting this clarification in here for future reference and resolving the comment for now.

Reply:
lol, I'm a dork.  Yes, "Azure Button" not "Azure Bastion"  

Had conversation offline.  I had a button-based deployment here, but decided to yank it because of subsequent steps that depended on the deployment name.  Plus it was becoming "option heavy."